.copr/Makefile

```diff
@@ -1,47 +1,19 @@
 # Copr does not have Git available. Install it before anything else.
```

Comment:
I guess this comment can be removed then

Reply:
Removed.